float32 goal_x
float32 goal_y
bool bump_observed
---
bool success
